connectivity/drivers/ble/FEATURE_BLE/TARGET_STM32WB/CMakeLists.txt

```diff
@@ -1,13 +1,15 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-target_include_directories(mbed-ble
+add_library(mbed-ble-stm32wb INTERFACE)
```

Comment:
Here for example, why isn't this library an `OBJECT` lib?

Reply:
I'll add a comment to main CMake file about how drivers are attached to the mbed-ble for now to explain it in one place.

Reply:
Done, also provided details in the commit message.